docs/server/configuration.md

```diff
@@ -0,0 +1,68 @@
+# Server Configuration
+
+The configuration of the software is done in the client and [campaign settings](/docs/dm/settings/).
+While you actually could modify the configuration stored in the save file (see below), it is not recommended and, thus, not documented, here.
+In contrast, this section focuses on the configuration that should be done within the central configuration file, which is `./server/server_config.cfg`.
+
+The configuration file is divided up into two sections: Webserver, and General.
+These sections are indicated by these designations, within square brackets (e.g. `[Webserver]`).
+Within each section, there are certain variables defined via a name and their value, using the scheme `name = value`, followed by a line-break.
+A leading `#` denotes a comment; every line with a leading `#` will be ignored.
+
+Only change the configuration if something does not work accordingly and you know what you're doing.
+
+## Webserver
+
+This section contains several variables concerning the webserver that distributes the client to the players.
+
+### host
+`host` (active by default, default: `0.0.0.0`, accepting IP addresses) tells the server on what IP address it is reachable.
+The default is set to `0.0.0.0` which, in this context, means: 'Any IP on this machine'.
+
+Most likeley, you won't need to change this setting, unless you're sharing the machine with others (e.g. on a shared (v)server) or run several servers on the machine with different addresses.
+
+### port
+`port` (active by default, default: `8000`, accepting numbers from 1 to 2^16 -1) tells the server on which port to listen.
+The default is set to `8000`, a [registered port commonly used for the Django Development Webserver](https://en.wikipedia.org/wiki/List_of_TCP_and_UDP_port_numbers#Registered_ports).
+You can freely adjust this setting according to your server's setup and will need to, if another application already listens on `8000`).
+In case you need to change the port, you might consider to choose a private port (>`49151`).
```

Comment:
I feel like this goes into too much detail.  The ip range, the registered ports are all unnecessary in my opinion.  We're not here to teach somebody what a port is, if they want to find out more they can find info very easily by googling this.

Also the recommendation for a port >49151 is a bit odd to me.  Most people will have almost no open ports and the ports they have open are typically known.

Keep it simple, this is the port, 8000 is the default

Reply:
As you wish. Unless there are concerns, I'll keep in the range of theoretically valid ports to keep the "tell default and valid input"-style of the docs. If there are concerns, just edit them out. It won't hurt me. :)